components/ItemGrid.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div>
+    <v-row v-if="loading">
+      <v-col cols="12" class="card-grid-container">
+        <skeleton-card v-for="n in 24" :key="n" />
+      </v-col>
+    </v-row>
+    <dynamic-scroller
+      v-if="items.length"
+      class="scroller"
+      :items="itemsChunks"
+      :min-item-size="350"
+      :buffer="$vuetify.breakpoint.height * 1.5"
+      page-mode
+    >
+      <template #default="{ item, index, active }">
+        <dynamic-scroller-item
+          :item="item"
+          :active="active"
+          :data-index="index"
+          class="card-grid-container"
+        >
+          <card v-for="card of item.chunk" :key="card.Id" :item="card" />
+        </dynamic-scroller-item>
+      </template>
+    </dynamic-scroller>
+    <v-row v-else-if="!loading" justify="center">
+      <v-col cols="12" class="card-grid-container empty-card-container">
+        <skeleton-card v-for="n in 24" :key="n" boilerplate />
+      </v-col>
+      <div class="empty-message text-center">
+        <slot>
+          <h1 class="text-h5">
+            {{ $t('noResultsFound') }}
+          </h1>
+        </slot>
+      </div>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { chunk } from 'lodash';
+import Vue from 'vue';
+
+export default Vue.extend({
+  props: {
+    items: {
+      type: Array,
+      required: true
+    },
+    loading: {
+      type: Boolean,
+      required: true
+    }
+  },
+  computed: {
+    itemsChunks: {
+      get() {
+        let cardsPerLine = 8;
+
+        if (this.$vuetify.breakpoint.smAndDown) {
```

Comment:
This seem to trigger a type error, but I'm not sure why (especially since it works properly).

Aside from this, this PR is ready to go.

Reply:
Shall we add Typescript ignore comments so it can still build?

Reply:
Yeah, I think it's okay for now. I'd prefer to properly fix it, but I have no idea why it does this...

Reply:
@MrTimscampi You don't specify the return type of the function inside the computed properties, so you're not referring to the Vue instance on this, rather on the ComputedProperties instance.

This is something I've seen we miss in this codebase and we should do a pass to verify that each function has it's return type defined correctly because it won't be long until they hit us like crazy with things like this.

I pushed a commit for fixing this, I hope you don't mind.